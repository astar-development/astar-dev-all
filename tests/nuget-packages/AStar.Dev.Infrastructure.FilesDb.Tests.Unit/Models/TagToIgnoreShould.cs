namespace AStar.Dev.Infrastructure.FilesDb.Models;

public sealed class TagToIgnoreShould
{
    [Fact]
    public void ReturnTheExpectedToStringOutput() =>
        new TagToIgnore().ToString().ShouldMatchApproved();
}
