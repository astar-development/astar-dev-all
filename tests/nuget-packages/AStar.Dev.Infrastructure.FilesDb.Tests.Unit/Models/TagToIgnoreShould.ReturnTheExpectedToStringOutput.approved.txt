{
  "id": 0,
  "value": "",
  "ignoreImage": false
}